docs/modeling-your-data/what-is-data-modeling/index.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: "What is data modeling?"
-date: "2020-10-30"
 sidebar_position: 0
 ---
 
-At Snowplow, we define data modeling as the process of using business logic to aggregate or otherwise transform event level data. It is the last step in the Snowplow pipeline, and occurs in the data warehouse.
+At Snowplow, we define data modeling as the process of using business logic to, transform, aggregate or otherwise process event level data. It occurs in the data warehouse, and is often they last step in the Snowplow pipeline (unless you are using reverse ETL tools).
```

Comment:
```suggestion
At Snowplow, we define data modeling as the process of using business logic to, transform, aggregate, or otherwise process event level data. It occurs in the data warehouse, and is often the last step in the Snowplow pipeline (unless you are using reverse ETL tools).
```

Reply:
Who doesn't love an oxford comma?

Reply:
You know, somehow it's never crossed my mind that it works with an `or`, I only ever thought about it with an `and` 🤯